Makefile

```diff
@@ -98,7 +98,7 @@ shared: metadata.db
 	sudo mkdir -p failed-image
 	sudo docker network inspect welder >/dev/null 2>&1 || sudo docker network create welder
 	sudo docker ps --quiet --all --filter 'ancestor=welder/bdcs-api-img' | sudo xargs --no-run-if-empty docker rm -f
-	sudo docker run -d --name api --restart=always -p 4000:4000 -v bdcs-recipes-volume:/recipes -v `pwd`:/mddb --network welder --security-opt label=disable welder/bdcs-api-img:latest
+	sudo docker run -d --name api --restart=always -v bdcs-recipes-volume:/recipes -v `pwd`:/mddb -v bdcs-socket:/run/weldr --security-opt label=disable welder/bdcs-api-img:latest
```

Comment:
I don't see where bdcs-socket comes from.  Regardless, I wouldn't even use the name "bdcs" in new stuff.  We're not using it right now.